radiation/get_data.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ -z "$IS_DOCKER" ] ; then
+    echo "Not in docker"
+else
+    gsutil cp -r gs://vcm-fv3gfs-serialized-regression-data/physics/fv3gfs-fortran-output/LW /work/radiation/fortran/data/.
+    gsutil cp -r gs://vcm-fv3gfs-serialized-regression-data/physics/lookupdata/lookup.tar.gz /work/radiation/python/lookupdata/.
+    gsutil cp gs://vcm-fv3gfs-serialized-regression-data/physics/standalone-output/LW/* /work/radiation/fortran/radlw/dump/.
+
+    cd /work/radiation/fortran/data/LW
+    tar -xzvf data.tar.gz
+
+    cd /work/radiation/python/lookupdata
+    tar -xzvf lookup.tar.gz
+
+    cd /work/radiation/fortran/radlw/dump
+    tar -xzvf data.tar.gz
```

Comment:
You have this in the jenkins action as well -- the jenkins action could call this script instead of duplicating the data grab.

Reply:
same comment as in for the jenkins action -- you might run into trouble if this is called more than once